### ISSUE TITLE
Question: Possible to override language list? 

### ISSUE THREAD

**Issue by dcworldwide** [i18next/i18next-browser-languageDetector#163]
Apologies, I can't find an issue for this (or a better place to ask).

TLDR; I want to reduce the number of HTTP requests sent to my server by i18n-next, when it looks for the locale json file(s) to use. 

Details...
This lib tries to resolve country specific variants of a language first, before defaulting to the standard definition. I.e en-GB or en-US are attempted first, followed by en. In my app, I will never have country specific variants, so requests looking for these locale file variants are wasteful and cause jank within my react views on slow bandwidth networks. Is it possible to configure this lib to resolve en-GB and en-US as en and never check the server for the country specific variants?

**Comment by jamuhl** [comment]
https://www.i18next.com/overview/configuration-options#languages-namespaces-resources -> load option -> set `load: 'languageOnly'` on init

**Comment by jamuhl** [comment]
If you like this module don’t forget to star this repo. Make a tweet, share the word or have a look at our https://locize.com to support the devs of this project -> there are many ways to help this project :pray:

**Comment by dcworldwide** [comment]
Thankyou sir.  Stared.
